--- openssl-1.0.1m.orig/apps/openssl.c	2016-11-28 14:01:12.606180466 +0000
+++ openssl-1.0.1m/apps/openssl.c	2016-11-28 14:01:12.786179694 +0000
@@ -269,6 +269,15 @@ int main(int Argc, char *ARGV[])
         for (i = 0; i < Argc; i++)
             Argv[i] = _Argv[i];
         Argv[Argc] = NULL;      /* Certain NULL termination. */
+<<<<<<<
+=======
+        free_Argv = 1;
+    } else {
+        /*
+         * Use the known-good 32-bit argv[] (which needs the type cast to
+         * satisfy the compiler), or the trusted or tested-good 64-bit argv[]
+         * as-is.
+>>>>>>>
         free_Argv = 1;
     } else {
         /*
